src/DotVVM.Framework/Hosting/DotvvmRequestContext.cs

```diff
@@ -104,9 +104,6 @@ public class DotvvmRequestContext : IDotvvmRequestContext
         /// </summary>
         public bool IsInPartialRenderingMode => DotvvmPresenter.DeterminePartialRendering(HttpContext);
 
-        [Obsolete("Get the IViewModelSerializer from IServiceProvider")]
-        public IViewModelSerializer ViewModelSerializer => Services.GetRequiredService<IViewModelSerializer>();
```

Comment:
I'm not sure, but this is IMO also a v2.0 obsolete.

Reply:
Yes, it's v2.0 obsolete. We should keep it.